### ISSUE TITLE
Library not working on latest android versions

### ISSUE THREAD

**Issue by MadhavNimavat** [kirillzyusko/react-native-wifi-p2p#98]
I have created a demo based on the code given and when I'm trying to run on Android 13 it not working as expected getAvailablePeers() function always returns an empty array. it not finding devices where I have two emulators .

**Comment by kirillzyusko** [comment]
Hey @MadhavNimavat 

Did you read [caveats](https://github.com/kirillzyusko/react-native-wifi-p2p?tab=readme-ov-file#caveats) section?

**Comment by c-info** [comment]
yes @kirillzyusko I have updated all required code also I'm trying to run the demo code provided in the readme

**Comment by kirillzyusko** [comment]
@c-info the example project has out-of-date code and I wrote it only when Android 9 was available. You'll need to modify the code yourself and ask all necessary permissions 🤷‍♂️

**Comment by c-info** [comment]
yes i have added all necessary permissions. Also let me know will it work by using two emulators ?

**Comment by kirillzyusko** [comment]
> Also, let me know if it will work by using two emulators.

Nope, never - only real devices and I wrote about that in the beginning of the README.

Please, provide adb logs, so that I can assist further
